recipes/restinio/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.6.7.1":
+    url: https://github.com/Stiffstream/restinio/archive/v.0.6.7.1.tar.gz
+    sha256: 68e26698a5463358f2e080fd6a7b07a8aab296c5192fbaed56a0a6a92ba98a12
```

Comment:
Should wait for restinio/0.6.8 (https://github.com/conan-io/conan-center-index/pull/1635#issuecomment-628387328).